apps/src/templates/amazonFutureEngineerEligibility.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {FormGroup, Button} from 'react-bootstrap';
+import FieldGroup from '../code-studio/pd/form_components/FieldGroup';
+import SchoolAutocompleteDropdownWithLabel from '@cdo/apps/templates/census2017/SchoolAutocompleteDropdownWithLabel';
```

Comment:
Might be worth moving that component to a more common location?

Reply:
I'm sure you're right -- I honestly do not understand what should go where in apps, if you'd be up to give me a rundown. I just wanted to at least separate this into a couple of files to start, and figure out where to put it later.